### ISSUE TITLE
I can't select pre-selected filter operator "contains"

### ISSUE THREAD

**Issue by schiessle** [nextcloud/tables#1235]
### Steps to reproduce

1. Go to a table
2. select the "..." menu next to a column header
3.  click "select operator"
4. "Contains" is pre-selected
5. I can't continue to the input field to enter my search term

### Expected behavior

I can click on "Contains" even that it is already selected and get forwarded to the next step to enter my search term. Or even something more intuitive, not everyone might try to click on a already selected option? cc @designer

### Actual behavior

Nothing happens if I click on "contain". I first have to chose a different operator, than click on the "Back" button and than I can select the operator "contains".

Tested with latest tables on c.nc.com

### Web server

None

### PHP engine version

None

### Database

None

**Comment by schiessle** [comment]
Now while writing the detailed steps I realized the solution is not to click on "select operator" but directly on "select value".  :see_no_evil: 

Feel free to close it again, or come up with some improvements of the flow. The fact that I didn't found the option for weeks might be a indicator that there is room for improvements? I will leave it to you, what you want to do with the ticket. Everything is fine for me.

**Comment by jancborchardt** [comment]
I agree it’s a bit confusing, but mostly cause you first have to go into a menu to select the operator, and (if operator is already the one you want) then back to select the value.

Quick fix: Also go to the "Select value" menu if you click on the already selected operator.

Ideal fix: Combine the "Select operator" and "Select value" views together since they belong together. Probably this should be done in a modal.

**Comment by enjeck** [comment]
should be fixed now
